### PR TITLE
Limit doubling time to smoothed line

### LINKED ISSUE #123: Improve doubling time plots: limit maximum of the y-axis to some fixed value?
Screenshot 2020-10-14 at 08 31 51

In the example above, there are some very large values, which make the interested line hard to read. An easy to implement fix would be to use a maximum value for either scale of at most 1000 days (or a similar number). 

Or introduce this maximum as a parameter in the function call, so that one can create one's own plots quite easily where this number can be changed?

### PR DESCRIPTION
**Pull request by kirienko**
We can solve use a decorator to set the Y-limit.

Also, I created a new file `plotting_helpers.py`, because `oscovida.py` is embarrassingly big already.

Closes #123

### CODE REVIEW

**Comment by kirienko** [comment]
Strangely enough, tests do well on my local machine :man_shrugging:

File: oscovida/plotting_helpers.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    for other_ax in ax.figure.axes:
+        if other_ax is ax:
+            continue
+        if other_ax.bbox.bounds == ax.bbox.bounds:
+            return True
+    return False
+
+
+def limit_to_smoothed(func, max_ratio=1.5):
+    def wrapper(*args, **kwargs):
```
Comment (ebadkamil) [inline oscovida/plotting_helpers.py:33]:
```
from functools import wraps

def limit_to_smoothed(func, max_ratio=1.5):
    @wraps(func)
    def wrapper(*args, **kwargs):
```

Comment (kirienko) [reply oscovida/plotting_helpers.py:33]: Fully agree, thanks!

**Comment by fangohr** [comment]
Does this need further review or can we merge it?

**Comment by fangohr** [comment]
Yes, needs further work: failure in https://travis-ci.com/github/oscovida/oscovida/jobs/404330728#L600

**Comment by kirienko** [comment]
Yes, exactly. There is still the case with `twinx`, that's why I put `[WIP]` status.

**Pull request merged**